### DISCUSSION TITLE
You attempted to use a firebase module that's not installed natively on your iOS

### DISCUSSION THREAD

**Discussion by GeoffreyOlivier** [infinitered/ignite#2537] [Q&A] [answered]
Hello, I use firebase for Auth, database, but for storage, I have an error only on IOS. My code work great on android.

There the error :

```
Error: You attempted to use a firebase module that's not installed natively on your iOS project by calling firebase.storage().

Ensure you have either linked the module or added it to your projects Podfile.

See http://invertase.link/ios for full setup instructions.
Error: You attempted to use a firebase module that's not installed natively on your iOS project by calling firebase.storage().

Ensure you have either linked the module or added it to your projects Podfile.

```

I don't know what to do. Can you please help me ? 

Thanks

**Comment by jamonholmgren** [comment]
@GeoffreyOlivier Did you get this figured out?

**Reply by GeoffreyOlivier** [reply, marked as answer]
I don't know, I start on a new emulator ios (iphone 14) and I don't have this problem. 
Option 1: the emulator iphone SE 3gen, is not compatible
Option 2: it's problem of cache
Option 3: I don't know
